controller.go

```diff
@@ -357,6 +357,10 @@ func (c *controller) NewNetwork(networkType, name string, options ...NetworkOpti
 		return nil, err
 	}
 
+	if network.Type() == "bridge" && network.isolated {
+		return nil, ErrNetworkNotSupportIsolation(network.Type())
```

Comment:
The isolate option could be passed in the driver network options, then parse in here
https://github.com/docker/libnetwork/blob/master/drivers/bridge/bridge.go#L475

Then 
https://github.com/docker/libnetwork/blob/master/drivers/bridge/bridge.go#L153 would fail with Not Implemented error until bridge driver implements the functionality